mlflow/store/tracking/file_store.py

```diff
@@ -285,7 +285,7 @@ def search_experiments(
                 )
         filtered = SearchExperimentsUtils.filter(experiments, filter_string)
         sorted_experiments = SearchExperimentsUtils.sort(
-            filtered, order_by or ["last_update_time DESC"]
+            filtered, order_by or ["creation_time DESC", "experiment_id ASC"]
```

Comment:
Changing the default sort ordering to prevent list items from jumping around as experiments are updated. A consistent order by creation time is consistent with 1.x behavior. For old 1.x experiments that don't have creation time attributes, we apply an additional default ordering of `experiment_id ASC`, which was used in 1.x